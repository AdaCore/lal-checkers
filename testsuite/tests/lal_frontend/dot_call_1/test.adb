procedure Test is
   package O is
      type A is tagged record
        E : Integer;
      end record;

      function Get_E(X : A) return Integer;
   end O;

   package body O is
      function Get_E(X : A) return Integer is
      begin
         return X.E;
      end Get_E;
   end O;

   P : O.A;
   R : Integer := P.Get_E;
begin
   null;
end Ex1;
