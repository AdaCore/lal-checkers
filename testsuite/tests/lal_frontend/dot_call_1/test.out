Program:
  P#1 = Input()
  R#2 = <SubpBody ["Get_E"] 11:7-14:17>(P#1)
  end:
